### PR TITLE
CIRCSTORE-347: Add tlr feature toggle job interface to Module Descriptor

### PR DESCRIPTION
**Pull request by yusuf-murodov-epam**
[CIRCSTORE-347](https://issues.folio.org/browse/CIRCSTORE-347)

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -589,6 +544,47 @@
           "delay": "5"
         }
       ]
+    },
+    {
+      "id": "tlr-feature-toggle-job-storage",
```
Comment (alexanderkurash) [inline descriptors/ModuleDescriptor-template.json:549]: move it above the _timer interface please

File: ramls/examples/actual-cost-record.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
   ],
   "itemBarcode": "888",
   "loanType": "Can Circulate",
-  "effectiveCallNumber": {
+  "effectiveCallNumberComponents": {
```
Comment (alexanderkurash) [inline ramls/examples/actual-cost-record.json:17]: why?

File: ramls/examples/actual-cost-records.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
       ],
       "itemBarcode": "888",
       "loanType": "Can Circulate",
-      "effectiveCallNumber": {
+      "effectiveCallNumberComponents": {
```
Comment (alexanderkurash) [inline ramls/examples/actual-cost-records.json:19]: why?

File: ramls/examples/actual-cost-record.json (hunk 2 above)
Comment (yusuf-murodov-epam) [reply ramls/examples/actual-cost-record.json]: reverted back

File: ramls/examples/actual-cost-records.json (hunk 3 above)
Comment (yusuf-murodov-epam) [reply ramls/examples/actual-cost-records.json]: reverted back

**Pull request closed without merging**